docs/package/gluon-web-logging.rst

```diff
@@ -0,0 +1,5 @@
+gluon-web-logging
+=================
+
+The *gluon-web-logging* package add a new section to advanced settings,
+to allow GUI based configuration of an remote syslog server.
```

Comment:
"The *gluon-web-logging* package adds a new section to the advanced settings to allow GUI based configuration of a remote syslog server"